packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.test.ts

```diff
@@ -394,10 +394,10 @@ describe("DefaultVisualizers unit tests", () => {
 	});
 
 	it("SharedTree", async () => {
-		const factory = new TreeFactory({});
+		const factory = SharedTree.getFactory();
 		const builder = new SchemaFactory("DefaultVisualizer_SharedTree_Test");
 
-		const sharedTree = factory.create(new MockFluidDataStoreRuntime(), "test");
+		const sharedTree = factory.create(new MockFluidDataStoreRuntime(), "test") as ITree;
```

Comment:
Is this how it works for other DDS factories today? Consumers are forced to cast?

Reply:
Yes, if you're invoking the result of 'getFactory()' directly.

For example, you can see that SharedMap.getFactory() also returns an untyped IChannelFactory:
```ts
	public static getFactory(): IChannelFactory {
		return new MapFactory();
	}
```

No, if you use AzureClient / ContainerSchema, in which case no cast is necessary:

```ts
		it("preserves 'SharedTree' type", async () => {
			const { container } = await client.createContainer({
				initialObjects: {
					tree: SharedTree,
				},
			});

			// Ensure that the 'tree' API is accessible without casting or suppressing lint rules:
			const tree: ITree = container.initialObjects.tree;
```